src/widget/OktaSignIn.js

```diff
@@ -117,9 +116,17 @@ var OktaSignIn = (function () {
     require.config(requireConfig);
     OktaAuth = require('vendor/OktaAuth');
     Util = require('util/Util');
+    LoginRouter = require('LoginRouter');
+
 
     authClient = new OktaAuth({ uri: options.baseUrl, transformErrorXHR: Util.transformErrorXHR });
-    _.extend(this, getProperties(authClient, options));
+    _.extend(this, getProperties(authClient, options, LoginRouter));
+    _.extend(this, LoginRouter.prototype.Events);
+
+    // Triggers the event up the chain so it is available to the consumers of the widget.
+    this.listenTo(LoginRouter.prototype, 'pageRenderedEvent', function(className) {
```

Comment:
I think in this case we want to listen to everything (like what bubble will give us) and trigger everything. This'll make it so that we don't have to update this code when we add more events.